Dockerfile

```diff
@@ -1,11 +1,16 @@
-FROM ruby:2.4.2-jessie
+FROM ruby:2.6.3
 RUN apt-get update -y && apt-get install -y cmake
 
 # Set default locale for Ruby to avoid encoding errors
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
 WORKDIR /app
+
+RUN gem update --system
+RUN gem install bundler:1.17.2
+
 COPY . .
 RUN bundle install
-ENTRYPOINT ["circlemator"]
+
+ENTRYPOINT ["bundle exec circlemator"]
```

Comment:
Something changed in `2.6` and running the binstub without `bundle exec` produces this error:

```
Traceback (most recent call last):
	2: from /usr/local/bundle/bin/circlemator:23:in `<main>'
	1: from /usr/local/lib/ruby/site_ruby/2.6.0/rubygems.rb:305:in `activate_bin_path'
/usr/local/lib/ruby/site_ruby/2.6.0/rubygems.rb:286:in `find_spec_for_exe': can't find gem circlemator (>= 0.a) with executable circlemator (Gem::GemNotFoundException)
```

I tried updating rubygems and recreating the bundle but it didn't help. Any ideas why this happens?

Reply:
Also applies to `2.5`

Reply:
It might be related to `bundler`'s update to 2

Reply:
/shrug

Reply:
It might be because you're getting bundle >2 and it's strange?

Reply:
Turns out it is irrelevant to the bundler version. To dig a bit more I pulled the original docker image for `rainforestapp/circlemator` and updated rubygems from `2.7.4` to `3.0.4`, which produced the same error. `rake install:local` generates correct binstubs though so I was able to remove the `bundle exec` from the entrypoint in the final setup.